src/libadapters.js

```diff
@@ -1,6 +1,6 @@
 // Method names for these signatures must be in src/asyncify_imports.json.
 const SIGNATURES = [
-  'ipp', // xProgress
+  'ipp', // xProgress, xCommitHook
```

Comment:
`xCommitHook` is made up: I couldn't find an official name for this callback. (I figured this is more helpful than harmful.)

Reply:
That's fine.